docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Trigger Cloud Sync conditionally"
+linkTitle: "Trigger Sync"
+description: "Configure cloud sync to automatically capture data in the Viam app."
+weight: 36
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+# SME: Alexa Greenberg
+---
+
+If you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+
+This module allows you to configure Cloud Sync to occur only at a specific time frame by implementing a sensor, `naomi:sync-at-time:timesyncsensor`, that enables sync when within a specified timeframe and disables sync when outside that timeframe.
+
+To set up conditional syncing you need to:
+
+1. Create or use an exisitng module that implements a `sensor` component that uses your custom logic to determine when to enable and disable sync.
+2. Change the configuration of the data manager to enable selective sync.
+
+## Requirements
+
+Before configuring your sensor, you must [create a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine) and you also need to:
+
+1. Enable [data capture](https://docs.viam.com/data/capture/).
+2. Enable [cloud sync](https://docs.viam.com/data/cloud-sync/).
+
+## Add sensor to determine when to sync
+
+In this example, you will configure sync to only trigger during a specific time frame of the day using an existing module [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time).
+If you need to trigger sync based on a different condition, you need to create your own module and adjust the module logic accordingly.
+Additional examples are available in this [GitHub Repo](https://github.com/viam-labs/trigger-sync-examples-v2).
+
+To use [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time):
+
+1. Go to your machine's **Config** page and click **Add Component**.
+2. Then select the `sync-at-time:timesyncsensor` model from the [`sync-at-time` module](https://app.viam.com/module/naomi/sync-at-time).
+3. Click **Add module**, then enter a name for your sensor and click **Create**.
+   The sensor will return true and enable sync when in a specified time frame.
+4. To configure your timeframe, go to the new component panel and copy and paste the following attribute template into your sensor’s **Attributes** box:
```

Comment:
```suggestion
4. To configure your time frame, go to the new component panel and copy and paste the following attribute template into your sensor’s **Attributes** box:
```
[nit] For consistency. Spelled with two words in most of this doc, which Google tells me is the most common nowadays